Formula/hamlib.rb

```diff
@@ -1,18 +1,8 @@
 class Hamlib < Formula
   desc "Ham radio control libraries"
   homepage "https://hamlib.sourceforge.io/"
-  url "https://src.fedoraproject.org/repo/pkgs/hamlib/hamlib-1.2.15.3.tar.gz/3cad8987e995a00e5e9d360e2be0eb43/hamlib-1.2.15.3.tar.gz"
-  sha256 "a2ca4549e4fd99d6e5600e354ebcb57502611aa63c6921c1b8a825289833f75e"
-
-  bottle do
```

Comment:
please don't remove the bottle block

Reply:
Thanks Sean, new to Homebrew, first commit.  Do I need to change the bottle sha256 values or just leave as is?  I was going for minimal w my commit.  Thanks!

Reply:
Just leave it as is, and our CI and merging procedure will deal with it automatically!